### PR TITLE
Separating Interfaces File

### PR DESCRIPTION
**Pull request by TaylorSmith28**
Separated the pH probe, stir controller, temp sensor, temp controller, and pump from the interfaces

### CODE REVIEW

File: test/utils/devices/serial_mock_test.py [hunk 1]
```diff
@@ -1,80 +1,64 @@
...
 def test_serial_flush():
     """
-    Function to test mock serial flush
+    The function to test mock serial flush
     """
-    arduino = serial.Serial(
-        port=c.ARDUINO_PORT, baudrate=c.ARDUINO_BAUD, timeout=c.ARDUINO_TIMEOUT
-    )
+    arduino = create_arduino()
     arduino.flush()
```
Comment (prestoncarman) [inline test/utils/devices/serial_mock_test.py:56]: Should this test have an assert?

File: test/utils/devices/serial_mock_test.py [hunk 2]
```diff
@@ -1,80 +1,64 @@
...
 def test_serial_input_buffer():
     """
-    Function to test mock serial input buffer
+    The function to test mock serial input buffer
     """
-    arduino = serial.Serial(
-        port=c.ARDUINO_PORT, baudrate=c.ARDUINO_BAUD, timeout=c.ARDUINO_TIMEOUT
-    )
+    arduino = create_arduino()
     arduino.reset_input_buffer()
```
Comment (prestoncarman) [inline test/utils/devices/serial_mock_test.py:40]: Should this test have an assert?

File: test/utils/devices/serial_mock_test.py [hunk 3]
```diff
@@ -1,80 +1,64 @@
...
 def test_serial_output_buffer():
     """
-    Function to test mock serial output buffer
+    The function to test mock serial output buffer
     """
-    arduino = serial.Serial(
-        port=c.ARDUINO_PORT, baudrate=c.ARDUINO_BAUD, timeout=c.ARDUINO_TIMEOUT
-    )
+    arduino = create_arduino()
     arduino.reset_output_buffer()
```
Comment (prestoncarman) [inline test/utils/devices/serial_mock_test.py:32]: Assert?

File: test/utils/devices/syringe_pump_mock_test.py [hunk 4]
```diff
@@ -1,24 +1,34 @@
...
+from titration.utils.devices.syringe_pump_mock import Syringe_Pump
+from titration.utils.devices.liquid_crystal_mock import LiquidCrystal
+from titration.utils.devices import board_mock as board_class
 
-from titration.utils import constants, interfaces
-from titration.utils.devices import syringe_pump_mock as syringe
 
-
-def setup_module():
+def create_lcd():
```
Comment (prestoncarman) [inline test/utils/devices/syringe_pump_mock_test.py:10]: This test file mixes LCD with syringe. It seems those should be separated.

File: test/utils/devices/temperature_control_mock_test.py [hunk 5]
```diff
@@ -1,79 +1,91 @@
...
-    )
-    temperature_controller = temperature_control.Temperature_Control(board.D1, sensor)
+    """
+    The function to test updating the temperature controller
+    """
+    temperature_controller = create_temperature_controller()
 
     temperature_controller.update()
     time.sleep(1)
     temperature_controller.update()
```
Comment (prestoncarman) [inline test/utils/devices/temperature_control_mock_test.py:37]: assert? what is the test?

Same question for all tests in this file.

File: titration/utils/devices/ph_probe.py [hunk 6]
```diff
@@ -23,3 +23,8 @@ def set_gain(self, gain):
 
     def get_gain(self):
         return self.ads.gain
+
+    def read_raw_pH(self):
```
Comment (prestoncarman) [inline titration/utils/devices/ph_probe.py:27]: Is this just an alternate name for `voltage()`. Why not just change the name?

File: titration/utils/devices/temperature_probe.py [hunk 7]
```diff
@@ -22,3 +22,10 @@ def get_temperature(self):
 
     def get_resistance(self):
         return self.sensor.resistance
+
+    def read_temperature(self):
```
Comment (prestoncarman) [inline titration/utils/devices/temperature_probe.py:26]: What value does this function add?

Comment (prestoncarman) [reply titration/utils/devices/temperature_probe.py:26]: I see ... interfaces added this function. Lets address this in a future update. Please add an issue to remove this function?

File: titration/utils/devices/ph_probe.py (hunk 6 above)
Comment (prestoncarman) [reply titration/utils/devices/ph_probe.py:27]: I see ... interfaces added this function. Let's address this in a future update. Please add an issue to remove/rename this function?

Comment (TaylorSmith28) [reply titration/utils/devices/ph_probe.py:27]: I'll add an issue.

File: titration/utils/devices/temperature_probe.py (hunk 7 above)
Comment (TaylorSmith28) [reply titration/utils/devices/temperature_probe.py:26]: I'll add an issue

**Comment by TaylorSmith28** [comment]
I'll address the other comments about testing in our next meeting.

**Review by prestoncarman — APPROVED** [review]

**Pull request merged**